Doctrine/DBAL/Types/DateTimeImmutableMicrosecondsType.php

```diff
@@ -20,22 +20,32 @@
 
 class DateTimeImmutableMicrosecondsType extends VarDateTimeImmutableType
 {
-    public function getName()
+    /**
+     * @return string
```

Comment:
Again, the entire docblock is redundant as string is already specified in the method return type-hint